schema/codelists/milestoneType.csv

```diff
@@ -7,4 +7,6 @@ delivery,Delivery milestones,"For delivery milestones, such as the date when a g
 reporting,Reporting milestones,"For reporting milestones, such as when key reports are to be provided."
 financing,Financing milestones,"For events related to the funding of the contracting process. In public-private partnerships, such events might include, for example, planned payments and equity transfers."
 payment,Payment milestones,"For payment milestones, such as the date when a payment is expected to be made to a supplier. If the `status` of a contract's payment milestone is 'met', then the contract ought to have an entry for the payment under its `implementation.transactions`."
+procurement,Procurement,"Events taking place during the procurement that are not captured by other codes, such as additional enquiry periods or draft tender submissions in a competitive dialogue process"
```

Comment:
It could be codes (like in this list) or fields (like tender period, etc.).

```suggestion
procurement,Procurement,"For events taking place during the procurement that are not captured elsewhere, such as additional enquiry periods or draft tender submissions in a competitive dialogue process."
```